contracts/RibbonStakingRewards.sol

```diff
@@ -204,7 +204,7 @@ contract StakingRewards is
             "Provided reward too high"
         );
 
-        periodFinish = block.timestamp.add(rewardsDuration);
+        periodFinish = Math.max(block.timestamp, startEmission).add(rewardsDuration);
```

Comment:
make periodFinish more precise to be exactly 28 days from start emission instead of 28 days from whenever we call it next friday (should be right before).

Reply:
What if we called this after `startEmission` for whatever reason? What about just doing `startEmission.add(rewardsDuration)` immediately?

Reply:
originally to allow notifyRewardAmount in the future after the rewardsDuration ends. but if we are positive, your way is better of course